packages/create-payload-app/src/lib/get-package-manager.ts

```diff
@@ -18,12 +18,16 @@ export async function getPackageManager(args: {
       detected = 'yarn'
     } else if (cliArgs?.['--use-npm'] || fse.existsSync(`${projectDir}/package-lock.json`)) {
       detected = 'npm'
+    } else if (cliArgs?.['--use-bun'] || fse.existsSync(`${projectDir}/bun.lockb`)) {
+      detected = 'bun'
     } else {
       // Otherwise check for existing commands
       if (await commandExists('pnpm')) {
         detected = 'pnpm'
       } else if (await commandExists('yarn')) {
         detected = 'yarn'
+      } else if (await commandExists('bun')) {
```

Comment:
I don't think bun should take precedence here. Let's remove.

Reply:
Done